src/cdk-experimental/testing/testbed/unit-test-element.ts

```diff
@@ -7,18 +7,48 @@
  */
 
 import {
-  dispatchFakeEvent,
-  dispatchKeyboardEvent,
+  clearElement,
   dispatchMouseEvent,
+  isTextInput, ModifierKeys,
```

Comment:
Just in case if you make changes to that file anyway: The import specifier is not aligned with others 😁